### PR TITLE
fix: updating end user page navigation (rhdevdocs-5564)

### PR DESCRIPTION
**Pull request by deerskindoll**
## What does this pull request change?
update of the end user navigation

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-5564

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by deerskindoll** [comment]
original issue https://github.com/eclipse/che/issues/22357

File: modules/hosted-che/nav.adoc
Comment (ibuziuk) [inline modules/hosted-che/nav.adoc:2]: @deerskindoll why do you want to remove `Try in Web IDE GitHub action` from menu ?

File: modules/end-user-guide/pages/getting-started-with-che.adoc
Comment (ibuziuk) [inline modules/end-user-guide/pages/getting-started-with-che.adoc:5]: is this comment expectected?

Comment (deerskindoll) [reply modules/end-user-guide/pages/getting-started-with-che.adoc:5]: the doc template requires it

File: modules/hosted-che/nav.adoc
Comment (deerskindoll) [reply modules/hosted-che/nav.adoc:2]: I didn't remove it. I moved it to the User guide.

**Review by ibuziuk — COMMENTED** [review]
I noticed that some of the sections are not named according to the https://github.com/eclipse/che/issues/22357
e.g. I can not find 'Starting a workspace from a Git repository URL`
@deerskindoll could you please make sure that all checkboxes from the upstream issue are addressed ? we also need @l0rd approval before we can merge
In general, I believe this refactoring is a good improvement for 3.9 👍

**Comment by deerskindoll** [comment]
> I noticed that some of the sections are not named according to the [eclipse/che#22357](https://github.com/eclipse/che/issues/22357) e.g. I can not find 'Starting a workspace from a Git repository URL` @deerskindoll could you please make sure that all checkboxes from the upstream issue are addressed ? we also need @l0rd approval before we can merge In general, I believe this refactoring is a good improvement for 3.9 👍

Thanks for pointing it out. I'll double-check. Consider this an evolving thing, I don't plan on merging this without @l0rd approving the changes.

**Review by l0rd — COMMENTED** [review]
@deerskindoll section "Starting a workspace from a raw devfile URL" should be at the same level as "Starting a workspace from a Git repository URL" whereas in your PR the latter includes the former.

**Review by l0rd — COMMENTED** [review]
Section "Using Che in team workflow" has 5 subsections but in the page there are only 2 links:
image

**Comment by deerskindoll** [comment]
> @deerskindoll section "Starting a workspace from a raw devfile URL" should be at the same level as "Starting a workspace from a Git repository URL" whereas in your PR the latter includes the former.

On it. Update: Done.

**Comment by deerskindoll** [comment]
> Section "Using Che in team workflow" has 5 subsections but in the page there are only 2 links: image

Let me get rid of the 3 pages. Update: Pages moved to their correct place.

**Review by l0rd — APPROVED** [review]
LGTM, thank you @deerskindoll!

**Pull request merged**
